scripts/build/build_node/Platform/Windows/install_python.ps1

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: Apache-2.0 OR MIT
 #>
 
 choco install -y python2 --version=2.7.15
```

Comment:
What do we have that still uses 2.7? Py 2.7 reached end of life January 1st, 2020

Reply:
I'm not sure, @amzn-changml might know why we need this on the Windows build nodes

Reply:
We don't need it anymore, it was a legacy of pre-build scripts that were originally in py2. That should be removed

Reply:
Ok, i'll remove it as part of this PR as well

Reply:
Thanks! If it's ok, can you also update the env var path in line 14 to reflect py 3.10's path?